README.md

```diff
@@ -1,55 +1,185 @@
+component is a component whose primary responsibility is to render some piece
+of the beautiful user interfaces that we get to build as web developers. Their
+job, in other words, is to look good!
+
+![I'm So Pretty](https://media.giphy.com/media/oLz0TmduZsUjm/giphy.gif)
+
+There is, however, a bit more thinking that we need to do here, just so we
+understand precisely what it means when we say that a component is of the type
+"presentational." Very often
+when we speak of classes or categories of things in world of programming, the types or
+classes that we are discussing are actually formalized in the libraries or
+languages themselves. Think, for example, of when we declare a React `Component.`
+When we do that we write either `React.createClass()` or (using the ES2015 format)
+`class SomeComponent extends Component`. Either way, we are creating an actual
+instance of `Component`.
+
+But here's the rub. When we say that some component is "presentational" *we
+definitely do not mean* that the component is a formal type or class defined
+by the React library. There is no such thing as type `PresentationalComponent`
+in the React library. Rather, what we are dealing with here is simply a useful
+*convention*, or, better yet, a *programming pattern* that coders who have used
+React have found themselves following over and over again as they compose
+their component-based React UIs.
+
+So what defines the presentational component pattern? Here's a list
+of defining features:
+* Presentational components are primarily concerned with how things look;
+* they probably only contain a render method;
+* they do not know how to load or alter the data that they render;
+* they rarely have any internally changeable `state` properties;
+* and, they are best written as stateless functional components.
+
+Okay so there's our pattern description. Now let's jump into some code and see
+how presentation components actually look in practice.
+
+## Surprise, You've already written presentational components!
+
+Yep, this is true. Think about it. If a presentational component is simply
+a component that doesn't know anything about how to get the data it displays,
+and is mainly responsible for presentation, then you've been doing this from
+the beginning. A simple `HelloWorld` component, for example, is almost
+certainly presentational. Let's see if that's right &mdash; we'll even give our
+component the ability to take a prop:
+
+```javascript
+const HelloWorld = React.createClass({
+  render: function() {
+    return <div class="hello-world">Hello {this.props.message || 'World' }</div>;
+  }
+})
+```
+
+Okay, so does this fit our pattern? Absolutely, it does. Here is a component
+that does nothing but render a piece of our UI; that has no notion of how
+to fetch or reload the `message` data that it takes in as a `prop`; that has
+no changeable state; and that only contains a render method. So, I think we can 
+safely say it's pretty well presentational.
```

Comment:
This is just a affectation. Was just playing with the language. But maybe it's too informal, and might be odd to non-native speakers. I'll fix.